src/config/dynamic-theme-fixes.config

```diff
@@ -26296,6 +26299,9 @@ img[src^="https://www.google.com/maps"]
 #shield-ok-fill-shield
 
 CSS
+.X6JNf:not(.NKrsie) {
+    background: var(--darkreader-background) !important;
```

Comment:
Did you check the fix. I'm afraid we don't have `--darkreader-background`, but `--darkreader-neutral-background`.